src/Kdyby/Doctrine/Proxy/ProxyAutoloader.php

```diff
@@ -51,6 +51,20 @@ public function __construct($proxyDir, $proxyNamespace)
 
 
 	/**
+	 * Returns an array of events this subscriber wants to listen to.
+	 *
+	 * @return array
+	 */
+	public function getSubscribedEvents()
+	{
+		return array(
+			'Nette\DI\Container::onInitialize' => 'register',
```

Comment:
onInitialize is called like this

```php
$this->getService('events.manager')->createEvent(array('Nette\\DI\\Container', 'onInitialize'))->dispatch($this);
```

that means the `$prepend` will be evaluated to TRUE. I'm not sure if that's desired.